File: src/main/java/com/plog/PlogApplication.java
package com.plog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.security.autoconfigure.SecurityAutoConfiguration;
import org.springframework.boot.security.autoconfigure.UserDetailsServiceAutoConfiguration;

@SpringBootApplication
public class PlogApplication {

    public static void main(String[] args) {
        SpringApplication.run(PlogApplication.class, args);
    }

}

File: src/main/java/com/plog/PlogApplication.kt
package com.plog

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object PlogApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(PlogApplication::class.java, *args)
    }
}
